Makefile

```diff
@@ -992,6 +992,9 @@ geojson/albers/state-bounds.json: geojson/albers/states.geojson
 geojson/albers/tile-index.json: geojson/albers/us-10m
 	cat $^/*.geojson | ./reproject-geojson --projection mercator --reverse | node_modules/.bin/tile-index -z 7 -f indexed > $@
 
+geojson/albers/centroid-%: geojson/albers/%
```

Comment:
@anandthakker I tried this, but polylabel choked on what appears to be a very simplified polygon that only had three points. Bumping it back up to the non-simplified geometries.